### PR TITLE
Add addEven function

### PR DESCRIPTION
**Pull request by ms10335**

### CODE REVIEW

**Comment by ziobron** [comment]
✅ CI Passed - Task: add-even
🏅 Granted 5 XP for @ms10335

**Comment by ziobron** [comment]
⏱ 3 XP extra for @ms10335 for on-time delivery

File: homework/add-even/addEven.hpp [hunk 1]
```diff
@@ -4,5 +4,20 @@
...
+        return returnEvenSum;
+    }
+    
+    for(auto it=numbers.begin(); it != numbers.end(); ++it)
+    {
+        if((*it & 1) ==0)
+        {
+            returnEvenSum += *it;
+        }
+    }
```
Comment (nauka-programowania-MA) [inline homework/add-even/addEven.hpp:19]: Add space between operations like '+' '-' '!=' '==' '%' etc...
Add space after while, if, else etc...

File: homework/add-even/addEven.hpp [hunk 2]
```diff
@@ -4,5 +4,20 @@
 int addEven(const std::vector<int>& numbers) {
     // TODO: Your implementation goes here
     // Below return is only to make this function compile now
-    return -1;
+    int returnEvenSum=0;
+    if(numbers.size() == 0)
+    {
+        return returnEvenSum;
+    }
```
Comment (nauka-programowania-MA) [inline homework/add-even/addEven.hpp:11]: redundant

File: homework/add-even/addEven.hpp [hunk 3]
```diff
@@ -4,5 +4,20 @@
 int addEven(const std::vector<int>& numbers) {
     // TODO: Your implementation goes here
     // Below return is only to make this function compile now
-    return -1;
+    int returnEvenSum=0;
```
Comment (nauka-programowania-MA) [inline homework/add-even/addEven.hpp:7]: Add space between operations like '+' '-' '!=' '==' '%' etc...
Add space after while, if, else etc...

**Review by nauka-programowania-MA — APPROVED** [review]

**Pull request closed without merging**